specs/components/dashboard/matches_section.spec.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
For the next iteration of this, should we add some mocks (sample data) to test that all the entries render, and that only certain entries show up when a specific category section is clicked on?

Reply:
Yes, definitely 👍